dotnet/src/webdriver/Interactions/WheelInputDevice.cs

```diff
@@ -224,23 +203,17 @@ public override Dictionary<string, object> ToDictionary()
 
             private Dictionary<string, object> ConvertElement()
             {
-                IWebDriverObjectReference elementReference = this.target as IWebDriverObjectReference;
-                if (elementReference == null)
+                if (this.target is IWebDriverObjectReference element)
                 {
-                    IWrapsElement elementWrapper = this.target as IWrapsElement;
-                    if (elementWrapper != null)
-                    {
-                        elementReference = elementWrapper.WrappedElement as IWebDriverObjectReference;
-                    }
+                    return element.ToDictionary();
                 }
 
-                if (elementReference == null)
+                if (this.target is IWrapsElement { WrappedElement: IWebDriverObjectReference wrappedElement })
```

Comment:
So modernized! I am old-school (I am not alone).

Reply:
I remember I used to jump on every modern innovation, and turn code into a mess just to use new features. These days I pick and choose.

I really like the property pattern matching personally, this method is now much shorter.

Reply:
It's simpler than it may seem

Before:

```csharp
if (this.target is IWrapsElement { WrappedElement: IWebDriverObjectReference wrappedElement })
{
}
```

After:

```csharp
if (this.target is IWrapsElement { WrappedElement: var wrappedElement })
{
    if (wrappedElement is IWebDriverObjectReference)
    {
        // current logic
    }
    else
    {
        // WrappedElement which is not IWebDriverObjectReference
    }
}
```

Reply:
Keep this example for another PR (about editorconfig), these changes are not related to the original PR.

Reply:
I understand the concern about maintainability. If this method may need to be re-written into an unwrapping loop or something.

In my opinion, the modernized code is easier to maintain compared to the current 20 lines of logic which took me a second to understand and unravel. The new pattern-matching declares itself directly: If `target` is an object reference, get its dictionary. If `target` wraps an object reference, gets the wrapped object reference dictionary. This is clearer to me, than the current code.

Reply:
I remember why I did this now! This was actually related to nullability, and I decided to use pattern-matching instead of adding `?` everywhere.  
![image](https://github.com/user-attachments/assets/342b1e6e-c09a-4e41-a9c8-6c2dc02aeff4)

Reply:
Reverted

Reply:
Thanks for understanding.